docs/content/install/install-on-bosh.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Install Athens with BOSH
+description: Installing an Athens Instance with BOSH
+weight: 11
```

Comment:
@s4heid I think this should go above the ["Managing private repos with .netrc files"](https://docs.gomods.io/install/managing-private-repos/) document, so the weight would need to be below 10. Are you ok with that?

Reply:
absolutely, I didn't know there is a limit. I changed it to 8, is that okay?